### PR TITLE
[ADD] Missing Batchnorm

### LINKED ISSUE #297: Batch normalization missing
I noticed that batch normalization is missing in the `MLP` and `ShapedMLP` backbone, we should probably add it to both implementations.

### PR DESCRIPTION
**Pull request by ravinkohli**
This PR fixes #297

### CODE REVIEW

**Review by nabenabe0928 — APPROVED** [review]
I do not see problems now.
